wagtailio/areweheadlessyet/models.py

```diff
@@ -2,11 +2,24 @@
 from wagtail.admin.edit_handlers import FieldPanel, MultiFieldPanel, StreamFieldPanel
 from wagtail.core.fields import RichTextField, StreamField
 from wagtail.core.models import Page
+from wagtail.snippets.models import register_snippet
 
 from wagtailio.areweheadlessyet.blocks import HomePageBlock
 from wagtailio.utils.models import CrossPageMixin, SocialMediaMixin
 
 
+@register_snippet
+class WagtailHeadlessIssue(models.Model):
+    number = models.PositiveIntegerField(primary_key=True)
+    title = models.CharField(max_length=255)
+    url = models.URLField()
```

Comment:
Question: The URL can always be derived from the issue number, is there a need to store it specifically?

eg:
```python
@property
def url(self):
    return f"https://github.com/wagtail/wagtail/issues/{self.number}/"
```

Reply:
I agree. I have changed it.